doc/troubleshooting.md

```diff
@@ -106,3 +110,19 @@ git push --tags --force
 ```
 
 See this [Stackoverflow post](https://stackoverflow.com/questions/5002555/can-a-lightweight-tag-be-converted-to-an-annotated-tag) for more details
+
+### Artifactory intergration issues
```

Comment:
This should be titled "Publishing to a private npm registry (Artifactory, npm Enterprise, etc)", as it isn't necessarily specific to Artifactory alone.